src/main/java/core/basesyntax/handlers/BalanceOperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.handlers;
+
+import core.basesyntax.db.Storage;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public void handle(String fruit,int quantity) {
+        Storage.fruits.put(fruit, quantity);
+    }
+}
```

Comment:
I think you should interact with your storage via StorageDao class object.
You make it a class field and initialize it in the constructor.
Then you will create it in Main and provide it to handlers as a parameter for filling your strategy map.
(That's for all handlers)

Reply:
In today's lection Sofia said thats not necessary bec we haven't study it yet